scope-validation-announcement.md

```diff
@@ -0,0 +1,10 @@
+# We are keep improving our service! But we might affect you , please read this.
```

Comment:
nit
"# We keep improving our service! But changes might affect you , please read this"